src/pages/Admin/Users/index.jsx

```diff
@@ -191,39 +222,103 @@ export default class Users extends Component {
     );
   };*/
 
-  renderCreateModal = () => {
+  renderOperateModal = () => {
     const { userStore, t } = this.props;
-    const { isCreateOpen, hideModal } = userStore;
+    const { userDetail, operateType, changeUser, changeUserRole } = userStore;
+
+    let title = t('Create New User');
+    if (operateType === 'modify') {
+      title = t('Modify User');
+    }
 
     return (
       <Modal
-        title={t('Create New User')}
-        visible={isCreateOpen}
-        onCancel={hideModal}
-        onOk={hideModal}
+        title={title}
+        visible={userStore.isCreateOpen}
+        onCancel={userStore.hideModal}
+        onOk={userStore.createOrModify}
       >
-        <div className="formContent">
-          <div className="inputItem">
-            <label>{t('Name')}</label>
-            <Input name="name" autoFocus maxLength="50" />
-          </div>
+        <form className="formContent">
+          {userDetail.user_id && (
+            <div className="inputItem">
+              <label>{t('Name')}</label>
+              <Input
+                name="name"
+                maxLength="50"
```

Comment:
`maxLength` should be number ?

Reply:
ok.maxLength=50 has error note,  maxLength= {50}

Reply:
right, expression placed in {}